internal/collector/sysinfo/platform/export_linux_test.go

```diff
@@ -34,3 +34,16 @@ func WithProStatusCmd(cmd []string) Options {
 		o.platform.proStatusCmd = cmd
 	}
 }
+
+// WithGetenv sets the getenv function for the linux platform collector using a map.
+func WithGetenv(env map[string]string) Options {
+	return func(o *options) {
+		o.platform.getenv = func(key string) string {
+			value, ok := env[key]
+			if !ok {
+				value = ""
+			}
+			return value
```

Comment:
since `env[key]` would return `""` if `key` was not present, could this be simplified to `return env[key]`?